src/js/utils/browser.js

```diff
@@ -4,7 +4,7 @@
 import document from 'global/document';
 import window from 'global/window';
 
-const USER_AGENT = window.navigator.userAgent;
+export const USER_AGENT = window.navigator.userAgent;
```

Comment:
Ah, ok. Any background on where this was used? It's not obvious to me why an alias to `userAgent` is helpful to outside code.